src/lib.rs

```diff
@@ -1247,75 +1834,52 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>>
         VOTE: VoteType<TYPES>,
     >(
         &self,
-        message: ConsensusMessage<TYPES, I>,
+        message: SequencingMessage<TYPES, I>,
     ) -> std::result::Result<(), NetworkError> {
         debug!(?message, "send_broadcast_message");
         self.inner
-            .quorum_exchange
+            .exchanges
+            .quorum_exchange()
             .network()
             .broadcast_message(
                 Message {
                     sender: self.inner.public_key.clone(),
-                    kind: message.into(),
+                    kind: MessageKind::from_consensus_message(message),
+                    _phantom: PhantomData,
                 },
                 // TODO this is morally wrong!
```

Comment:
I think I wrote this comment. No idea why I thought this was morally wrong, but it's probably fine and worth removing the comment?

Reply:
Yeah I think we could remove it!